modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -6075,9 +6075,15 @@ protected String addImageFileEntries(JournalArticle article, String value)
 					folder.getGroupId(), folder.getFolderId(),
 					tempFileEntry.getFileName(), false);
 
+				// The UUID of the temporary file is stored on the page.
+				// However, when the system auto saves, this temporary file will
+				// be deleted. To ensure we can still identify the file, we will
+				// use the temporary file's UUID as the ERC for the final file.
+				// See LPD-52357.
+
```

Comment:
I've sent a proposal to change this.

The UUID of the temporary file is stored on the field once saved as temp.
However, when the system auto saves, this temporary file will
be deleted and it's published with a different UUID, so it doesn't match. To been able to use the file, we'll
using the temporary file's UUID as published file entry ERC so we could fetch it using this field.
See LPD-52357.

Reply:
cahnged